core/storage/email/gae_models_test.py

```diff
@@ -244,30 +244,6 @@ def test_get_multi_by_user_ids_works_correctly(self):
         self.assertIn('user_id_1', result.keys())
         self.assertIn('user_id_2', result.keys())
 
-    def test_user_id(self):
-        result = email_models.GeneralFeedbackEmailReplyToIdModel.create(
-            'user_id', 'entity_type.entity_id.thread_id')
-
-        self.assertEqual(result.user_id, 'user_id')
```

Comment:
The result key is of the type `google.appengine.ext.ndb.key.Key` (https://cloud.google.com/appengine/docs/standard/python/refdocs/google.appengine.ext.ndb.key). The model you need can be obtained using `result.key.pairs()[0]`.
This provides a tuple for the model: `('GeneralFeedbackEmailReplyToIdModel', 'user_id.entity_type.entity_id.thread_id')`

Now you can use the 2nd element of the tuple to assert whether the desired format is obtained or not.

```
    def test_user_id(self):
        result = email_models.GeneralFeedbackEmailReplyToIdModel.create(
            'user_id', 'entity_type.entity_id.thread_id')
        result_instance = result.key.pairs()[0]
        user_id = result_instance[1].split('.')[0]
        self.assertEqual(user_id, 'user_id')
```

Reply:
I see ! Thanks, fixed.